packages/compass-crud/src/components/table-view/cell-renderer.jsx

```diff
@@ -238,9 +237,9 @@ class CellRenderer extends React.Component {
       return null;
     }
     return (
-      <div className={'table-view-cell-circle-button'} onClick={this.handleDrillDown.bind(this)}>
+      <button className={BUTTON_CLASS} onClick={this.handleDrillDown.bind(this)}>
```

Comment:
Updated the `div` here to be an interactive html element (and the button above as well.

Here's how they look (same as before):
<img width="182" alt="Screen Shot 2022-05-19 at 10 12 21 PM" src="https://user-images.githubusercontent.com/1791149/169452011-a46115f8-4c3d-4203-8cd0-8a48eb5aa89b.png">
<img width="218" alt="Screen Shot 2022-05-19 at 10 13 18 PM" src="https://user-images.githubusercontent.com/1791149/169452012-2f5ff08c-13d1-4e9d-9937-08e9e9b3e65b.png">